### PR TITLE
Profiled windows builds

### PR DESCRIPTION
**Pull request by jbgi**

### CODE REVIEW

File: nix/haskell.nix [hunk 1]
```diff
@@ -46,8 +46,12 @@ let
         packages.ekg.components.library.enableSeparateDataOutput = true;
         packages.cardano-node.configureFlags = [ "--ghc-option=-Werror" ];
         packages.cardano-config.configureFlags = [ "--ghc-option=-Werror" ];
-        enableLibraryProfiling = profiling;
       }
+      (lib.optionalAttrs profiling {
+        enableLibraryProfiling = true;
+        packages.cardano-node.enableExecutableProfiling = true;
+        profilingDetail = "all-functions";
```
Comment (dcoutts) [inline nix/haskell.nix:53]: I think "exported-functions" should be enough. The more detail we include the slower the code runs, and the more excess detail we get in the final profiles, which can make it hard to see what is going on.

Comment (jbgi) [reply nix/haskell.nix]: fixed (`exported-functions` is the default)

**Comment by jbgi** [comment]
`nix build -f release.nix cardano-node-win64-profiled` [currently fails](https://iohk-nix-cache.s3-eu-central-1.amazonaws.com/log/4ravpwdbh6dbdqxmwj1hhi0f9vjbag59-x86_64-w64-mingw32-streaming-bytestring-0.1.6-lib-streaming-bytestring-x86_64-w64-mingw32.drv) with
```
<no location info>: error:
    /nix/store/ghcll0zjzfn1ss2x03msjclfv1vb8zm5-iserv-wrapper/bin/iserv-wrapper-prof: createProcess: runInteractiveProcess: exec: does not exist (No such file or directory)
```
I could not find from where this non-existing `iserv-wrapper-prof`is called from.
Only `iserv-wrapper` is present in that directory.

@angerman would you have some more insight into this error?

**Comment by jbgi** [comment]
closing for now. More investigation on issues of the compilation pipeline is required.

> Further investigation shows that we are failing to malloc in the linker, looking at the wine process, I don't see it running out of memory (and the machine has 64G). Not clear yet what's really the issue with profiling here, however profiling libraries are usually larger (+50%) than regular libraries

**Pull request closed without merging**